test/workers/repository/master-issue.spec.ts

```diff
@@ -1,21 +1,22 @@
-const fs = require('fs');
-const masterIssue = require('../../../lib/workers/repository/master-issue');
+import fs from 'fs';
+import { mock } from 'jest-mock-extended';
 
-/** @type any */
-const { platform } = require('../../../lib/platform');
+import * as masterIssue from '../../../lib/workers/repository/master-issue';
+import { RenovateConfig, getConfig, platform } from '../../util';
```

Comment:
@ViceIce Any ideas about this one? All tests pass when run individually, but only the first 2 pass when run as a suite. Something up with resetting the mocks, but I'm not sure exactly what's going wrong.

Reply:
See below